app/src/routes/project/LayersPanel/index.tsx

```diff
@@ -74,17 +78,26 @@ const LayersPanel = observer(() => {
     return (
         <div
             className={clsx(
-                'border min-w-60 max-w-96 bg-black/80 backdrop-blur rounded-tr-xl shadow',
```

Comment:
This used to have `min-w-60 max-w-96` which would limit the amount you can resize the panel. This is so that when resized, the panel doesn't become too small or too large. This is a regression.